### PR TITLE
Value Set and Code system for Place of Service

### PR DESCRIPTION
**Pull request by Gokulsignal**
@enjoysparkling  Value Set and Code system for Place of Service is updated. Kindly review.

### CODE REVIEW

File: input/vocabulary/codesystem-place-of-service.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    "status": "draft",
+    "experimental": false,
+    "date": "2023-08-16T16:08:23+05:30",
+    "description": "CMS maintains a list of Place of Service (POS) codes that indicate where a service can be provided.",
+    "caseSensitive": true,
+    "valueSet": "https://signalbhn.org/fhir/ValueSet/place-of-service",
+    "content": "complete",
+    "concept": [
+        {
+            "code":"2",
```
Comment (enjoysparkling) [inline input/vocabulary/codesystem-place-of-service.json:16]: Need to retain Leading Zeroes

File: input/vocabulary/codesystem-place-of-service.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
+{
+    "resourceType": "CodeSystem",
+    "id": "place-of-service",
+    "url": "https://signalbhn.org/fhir/CodeSystem/place-of-service",
+    "name": "PlaceOfService",
+    "title": "Place Of Service",
+    "status": "draft",
+    "experimental": false,
+    "date": "2023-08-16T16:08:23+05:30",
+    "description": "CMS maintains a list of Place of Service (POS) codes that indicate where a service can be provided.",
```
Comment (enjoysparkling) [inline input/vocabulary/codesystem-place-of-service.json:10]: Add this URL to the description:  https://www.cms.gov/medicare/coding-billing/place-of-service-codes/code-sets

File: input/vocabulary/codesystem-place-of-service.json
Comment (enjoysparkling) [inline input/vocabulary/codesystem-place-of-service.json:1]: Update this code system to match https://www.cms.gov/medicare/coding-billing/place-of-service-codes/code-sets.

File: input/vocabulary/valueset-place-of-service.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-place-of-service.json:1]: See changes on related codesystem.

File: input/vocabulary/valueset-place-of-service.json [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+            "telecom": [
+                {
+                    "system": "url",
+                    "value": "http://hl7.org/fhir"
+                }
+            ]
+        }
+    ],
+    "description": "CMS maintains a list of Place of Service (POS) codes that indicate where a service can be provided.",
+    "compose": {
```
Comment (enjoysparkling) [inline input/vocabulary/valueset-place-of-service.json:41]: No need to list out every code if you are including the entire code system.  Just use the include system, e.g.
```
  "compose" : {
    "include" : [{
      "system" : "http://terminology.hl7.org/CodeSystem/ex-serviceplace"
    }]
  }
```

**Review by enjoysparkling — CHANGES_REQUESTED** [review]

File: input/vocabulary/valueset-place-of-service.json
Comment (enjoysparkling) [reply input/vocabulary/valueset-place-of-service.json:1]: ALSO - do not use the HL7 example code system. Just copy the CMS values and use that directly.
verify the CMS POS values align with the BHA USCS.

File: input/vocabulary/codesystem-place-of-service.json (hunk 1 above)
Comment (Gokulsignal) [reply input/vocabulary/codesystem-place-of-service.json:16]: Retained the Leading Zeroes, please review @enjoysparkling

File: input/vocabulary/codesystem-place-of-service.json
Comment (Gokulsignal) [reply input/vocabulary/codesystem-place-of-service.json:1]: The code system is matched with the POS details given in the URL. NA ranges are included as per the table.

File: input/vocabulary/valueset-place-of-service.json
Comment (Gokulsignal) [reply input/vocabulary/valueset-place-of-service.json:1]: The changes made on the code system is updated accordingly to the valueset.

File: input/vocabulary/valueset-place-of-service.json (hunk 3 above)
Comment (Gokulsignal) [reply input/vocabulary/valueset-place-of-service.json:41]: Just included the system to the valueset file. Kindly review @enjoysparkling

File: input/vocabulary/codesystem-place-of-service.json
Comment (enjoysparkling) [reply input/vocabulary/codesystem-place-of-service.json:1]: I would also like to see the [CodeSystem.concept.definition](http://hl7.org/fhir/codesystem-definitions.html#CodeSystem.concept.definition) populated from the [CMS POS](https://www.cms.gov/medicare/coding-billing/place-of-service-codes/code-sets).

Is there a reason it was not included?

**Review by enjoysparkling — DISMISSED** [review]

**Review by arunsundar-m — APPROVED** [review]

**Review by arunsundar-m — APPROVED** [review]

**Review by arunsundar-m — APPROVED** [review]

**Comment by arunsundar-m** [comment]
@jfabrizio329 Please approve the merge request.

**Review by arunsundar-m — APPROVED** [review]

**Review by vijay61 — APPROVED** [review]

**Pull request merged**